handler.js

```diff
@@ -27,19 +27,22 @@ const instance_regions = {
 // Welcome message to user (telegram)
 const initiate = {
   'en': 'Welcome! Type in /flood to request a card link',
-  'id': 'Selamat datang! ketik /banjir untuk meminta link kartu'
+  'id': 'Selamat datang! ketik /banjir untuk meminta link kartu',
+  'in': 'Hai!!  Saya BencanaBot. Tekan menu di bawah atau ketik ’Laporkan banjir'
 }
 
 // Replies to user
 const replies = {
   'en': 'Hi! Report using this link, thanks.',
-  'id': 'Hi! Laporkan menggunakan link ini. Terima kasih.'
+  'id': 'Hi! Laporkan menggunakan link ini. Terima kasih.',
+  'in': 'Hi! Laporkan menggunakan link ini. Terima kasih.'
```

Comment:
lets only keep en and id for the moment - will create India specific languages later (ta:tamil)

Reply:
done